test/bats/plugin-test.bats

```diff
@@ -81,8 +81,7 @@ SLEEP_TIME=1
     run kubectl replace -f ./config/samples/config_v1beta1_store_oras.yaml
     sleep 5
 
-    run kubectl run cosign-demo-keyless --namespace default --image=wabbitnetworks.azurecr.io/test/cosign-image:signed-keyless
-    assert_success
+    wait_for_process 20 10 'kubectl run cosign-demo-keyless --namespace default --image=wabbitnetworks.azurecr.io/test/cosign-image:signed-keyless'
```

Comment:
consider adding a retry message with outputs of the command

Reply:
consider updating the cosign verifier docs to mention potential timeout

Reply:
added note to the cosign doc. This doc should probably be migrated to the website.

added a retry statement with the command that gets outputed to a separate IO stream that is always printed. see docs: https://bats-core.readthedocs.io/en/stable/writing-tests.html#printing-to-the-terminal